TestFoundation/TestNSHTTPCookieStorage.swift

```diff
@@ -227,18 +227,22 @@ class TestNSHTTPCookieStorage: XCTestCase {
         storage.setCookie(testCookie)
         XCTAssertEqual(storage.cookies!.count, 3)
         var destPath: String
+        //let bundleName = Bundle.main.object(forInfoDictionaryKey: "CFBundleDisplayName") as! String
+        //TODO: Above api returns nil on Linux when this code is introduced. Hence, using the below workaround.
+        //Needs replacement once fixed.
+        let bundlePath = Bundle.main.bundlePath
+        var bundlePathComponents = bundlePath.components(separatedBy: "/")
+        let bundleName: String = "/" + bundlePathComponents[bundlePathComponents.count - 1]
```

Comment:
Searched around to check if there is any other way I can get the Product name details of the command line, but in vain. Any ideas please?

Yep, removing the explicit String. 

bundlePathComponents.last it is. Thanks!